src/main/resources/hudson/tasks/junit/pipeline/JUnitResultsStep/help-skipMarkingBuildUnstable.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    If this option is unchecked, then the plugin will mark the build as unstable when it finds at least 1 test failure.
+    If this option is checked, then the build will still be successful even if there are test failures reported.
+    In any case, the corresponding pipeline node (and stage) will be marked as unstable in case of test failure.
```

Comment:
is there existing test coverage for this already? otherwise could you add it please?

Reply:
In many tests, there is already `assertStageResults(r, 1, 8, 3, "first");` that is checking for this (existence of a `WarningAction` attached to the node)